src/testE2E/amazonq/featureDev.test.ts

```diff
@@ -0,0 +1,127 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as got from 'got'
+import assert from 'assert'
+import { qTestingFramework } from './framework/framework'
+import { FollowUpTypes } from '../../amazonqFeatureDev/types'
+import sinon from 'sinon'
+import { FeatureDevClient } from '../../amazonqFeatureDev/client/featureDev'
+import { verifyTextOrder } from './framework/text'
+import { examples } from '../../amazonqFeatureDev/text'
+
+describe('Test Amazon Q Feature Dev', function () {
```

Comment:
test names never need the word "test", "should", "correct", etc.
```suggestion
describe('Amazon Q Feature Dev', function () {
```